src/x_array.c

```diff
@@ -713,20 +713,23 @@ typedef struct _array_random   /* any operation meaningful on a subrange */
     unsigned int x_state;
 } t_array_random;
 
+int makeseed();
```

Comment:
This should be `int makeseed(void);`.

NOTE: in C++ it would be the same as `int makeseed(void);`, but in C this is the old K&R function declaration style without parameters.